src/login/login.js

```diff
@@ -199,6 +199,20 @@ export default function loginFactory($container, config) {
 
             hide();
 
+            $inputToggle.on('keypress', function(e) {
```

Comment:
A similar code exists to apply on the spacebar ([see line 224](https://github.com/oat-sa/tao-core-ui-fe/blob/3795ab7a19a285de046c2d17dd7edf718003ccaa/src/login/login.js#L224)).

A better approach would have bee to factorize the behavior:
```javascript
function togglePassword() {
    if ($pwdInput.type === 'password') {
        show();
    } else {
        hide(true);
    }
}

$inputToggle.on('keyup', function (e) {
    if (e.key === ' ') {
        togglePassword();
    }
});
$inputToggle.on('keypress', function (e) {
    if (e.key === 'Enter') {
        togglePassword();
        e.stopPropagation();
    }
});
```